test/index.js

```diff
@@ -6,21 +6,38 @@ var assert = require('assert');
 var Identify = facade.Identify;
 var Track = facade.Track;
 var Drip = require('..');
+var redis = require('redis');
 
-describe('Drip', function(){
+describe('Drip', function() {
   var settings;
   var payload;
   var test;
   var drip;
+  var db;
 
-  beforeEach(function(){
+  before(function(done) {
+    db = redis.createClient();
+    db.on('ready', done);
+    db.on('error', done);
+  });
+
+  beforeEach(function() {
```

Comment:
Could we add a `db.flushall` to the beforeEach just to make sure redis is empty when we're starting